rtlib/src/DisegniSegmenter.cpp

```diff
@@ -4,15 +4,27 @@
 #include <limits>
 #include <map>
 #include <set>
+#include <iostream>
```

Comment:
```suggestion
```

Reply:
Did you mean to leave this comment?

Reply:
Yes. You shouldn't need iostream once you remove the `std::cout` stuff

Reply:
Oh, sorry. No. I meant to remove the one in your class, not here.

Reply:
I don't know what's happening. GitHub keeps showing this under different files. Remove the iostream include and all uses of `std::cout` from your class.